src/common_node.cpp

```diff
@@ -4,13 +4,14 @@ using namespace common_lib;
 
 /**
  * @brief Callback function for the timer.
- * 
+ *
  * This function is called when the timer expires. It creates a Heartbeat message,
  * sets the necessary fields, and publishes the message using the publisher.
- * 
+ *
  * @return void
  */
-void CommonNode::heartbeat_timer_callback() {
+void CommonNode::heartbeat_timer_callback()
```

Comment:
Könnte doch eigentlich const sein.
```suggestion
void CommonNode::heartbeat_timer_callback() const
```

Reply:
Da in der function der heartbeat_tick hochgezählt wird ist das nicht möglich